gulpfile.js

```diff
@@ -27,7 +31,7 @@ require('gulp-task-loader')({
 // Define task for each theme, locale, lang, processing type etc.
 // Gulp can't run same task in parallel, so we need different names
 Object.keys(config.themes).forEach(name => {
-  var theme = config.themes[name];
+  const theme = config.themes[name];
   theme.locale.forEach(locale => {
     gulp.task(
       theme.lang + ':' + name + ':' + locale,
```

Comment:
Since config loader may not throw error's for themes.json at the time it's called we still need get-themes. I improved get themes to always support `--theme <theme name>`  It's got 2 error throw conditions but that's fine actually, since we still want to stop things like `gulp styles` that use themes.json when themes.json is not available. Also throwing an `plugins.util.PluginError` doesn't do well bringing the error while it's not used inside a gulp task. Which this resolves 👍  Let me know what your thoughts on this are. I am more than happy to improve it further, just can't see another way than returning an empty object when themes.json is not available ATM. The changes to config-loader will now allow it to fail on all other config loads.